frontend/ui/elements/screen_android.lua

```diff
@@ -5,9 +5,42 @@ local logger = require("logger")
 local _ = require("gettext")
 local Input = Device.input
 local Screen = Device.screen
+local T = require("ffi/util").template
 
 if not isAndroid then return end
 
+-- screen timeout
+
+local can_modify_timeout = not android.needsWakelocks()
+
+local timeout_system = 0	-- same as system (do nothing!)
+local timeout_custom1 = 30000	-- 30s
+local timeout_custom2 = 60000	-- 60s
+local timeout_custom3 = 120000	-- 2m
+local timeout_custom4 = 300000	-- 5m
+local timeout_custom5 = 600000	-- 10m
+local timeout_custom6 = 900000	-- 15m
+local timeout_custom7 = 1800000	-- 30m
```

Comment:
You could also consider writing 30 * 60 * 1000 if you think that's clearer. (luacheck would like there to be only spaces I'm afraid btw)